### PR TITLE
deliver initial crawler payload

### PR DESCRIPTION
**Pull request by DerekNonGeneric**

### CODE REVIEW

File: _data/stage3.yml [hunk 1]
```diff
@@ -238,3 +280,13 @@
...
     - Daniel Ehrenberg
+  description: A proposal for syntax to import ES modules with assertions.
+  example: >-
+    new Worker(&#x22;foo.wasm&#x22;, { type: &#x22;module&#x22;, assert: { type:
+    &#x22;webassembly&#x22; } });
+  has_specification: true
+  presented:
+    - date: "September\_2020"
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2020-09/sept-22.md#import-assertions-for-stage-3
```
Comment (DerekNonGeneric) [inline _data/stage3.yml:277]: 🔧 (notable correction) url to meeting notes for this proposal have been added

File: _data/stage3.yml [hunk 2]
```diff
@@ -91,143 +115,161 @@
...
+        Object.defineProperty(C.prototype, &#x22;at&#x22;,
+                              { value: at,
+                                writable: true,
+                                enumerable: false,
+                                configurable: true });
+    }
+  has_specification: true
+  presented:
+    - date: "November\_2020"
+      url: at
```
Comment (DerekNonGeneric) [inline _data/stage3.yml:268]: ⚠️

Comment (DerekNonGeneric) [inline _data/stage3.yml:268]:
```suggestion
      url: https://github.com/tc39/notes/blob/master/meetings/2020-11/nov-17.md#item-rename--revisit-inclusion-on-string
```

File: _data/stage3.yml [hunk 3]
```diff
@@ -91,143 +115,161 @@
...
+        Object.defineProperty(C.prototype, &#x22;at&#x22;,
+                              { value: at,
+                                writable: true,
+                                enumerable: false,
+                                configurable: true });
+    }
+  has_specification: true
+  presented:
+    - date: "November\_2020"
+      url: https://github.com/tc39/notes/blob/master/meetings/2020-11/nov-17.md#item-rename--revisit-inclusion-on-string
```
Comment (DerekNonGeneric) [inline _data/stage3.yml:268]: 🔧

**Review by chicoxyzzy — APPROVED** [review]

File: _data/stage3.yml [hunk 4]
```diff
@@ -1,88 +1,112 @@
...
+- id: proposal-regexp-legacy-features
   authors:
     - Claude Pache
   champions:
     - Mark Miller
     - Claude Pache
+  description: >-
+    This is a specification draft for the legacy (deprecated) RegExp features in
+    JavaScript, i.e., static properties of the constructor like RegExp.$1 as
+    well as the RegExp.prototype.compile method.
```
Comment (DerekNonGeneric) [inline _data/stage3.yml]: All data here is output from the crawler, but if we fix this once, the changes should persist on subsequent runs.
<br>
```suggestion
    JavaScript, i.e., static properties of the constructor like <code>RegExp.$1</code> as
    well as the <code>RegExp.prototype.compile</code> method.
```

File: _data/stage3.yml [hunk 5]
```diff
@@ -91,143 +115,161 @@
...
-  has_specification: true
-  description: A proposal for an "asynchronous atomic wait" for ECMAScript, primarily for use in agents that are not allowed to block.
+- id: proposal-atomics-wait-async
   authors:
     - Lars Hansen
   champions:
     - Shu-yu Guo
     - Lars Hansen
+  description: >-
+    A proposal for an "asynchronous atomic wait" for ECMAScript, primarily for
```
Comment (DerekNonGeneric) [inline _data/stage3.yml]:
```suggestion
    A proposal for an &ldquo;asynchronous atomic wait&rdquo; for ECMAScript, primarily for
```

File: _data/stage3.yml [hunk 6]
```diff
@@ -91,143 +115,161 @@
...
   authors:
     - Shu-yu Guo
     - Tab Atkins
   champions:
     - Shu-yu Guo
     - Tab Atkins
+  description: >-
+    A proposal to add a <code>.at()</code> method to all the basic indexable
+    classes (<code>Array</code>, <code>String</code>, <code>TypedArray</code>).
+  example: |-
```
Comment (codehag) [inline _data/stage3.yml:245]: This is the polyfill, not the example. The proposal doesn't have an example, and the one I entered manually should be used in this case.

**Comment by codehag** [comment]
Please update the title of this pr so it is more descriptive, and address the issue with the manually added information being lost. See https://github.com/tc39/tc39.github.io/pull/211#issuecomment-707558360 regarding this issue with the crawler.

File: _data/stage3.yml (hunk 1 above)
Comment (codehag) [reply _data/stage3.yml:277]: Emojis without context can be an accessibility issue. If you have a comment, please write it out so it is clear for others.

File: _data/stage3.yml [hunk 7]
```diff
@@ -91,143 +115,161 @@
...
+    }
+
+    // Other TypedArray constructors omitted for brevity.
+    for (let C of [Array, String, Uint8Array]) {
+        Object.defineProperty(C.prototype, &#x22;at&#x22;,
+                              { value: at,
+                                writable: true,
+                                enumerable: false,
+                                configurable: true });
+    }
```
Comment (DerekNonGeneric) [inline _data/stage3.yml]:
```suggestion
    var list = [&#x22;apple&#x22;, &#x22;orange&#x22;, &#x22;banana&#x22;];
    list.at(-1) // banana, using negative index
    list.at(0)  // apple
    list.at(1)  // orange
    list.at(2)  // banana
```

**Comment by DerekNonGeneric** [comment]
Titles purposely not &ldquo;update stage 3 January 2021&rdquo; due to there being a date format that looks like `DD Month YYYY`, which could end up looking like January 3rd, 2021. Will have to settle on something once these become automated, though.

**Comment by codehag** [comment]
A better title here would be something to the effect of "automated update of notes links" or "automated update of notes links for November 2020", as that is what this is doing. The date "January 2021" is misleading in a few ways. We will have a meeting in January 2021, but this commit doesn't have anything to do with it. The second is that this date information in this commit isn't important, as we already have that information in the commit itself.

Regarding the automation commit messages -- since you are suggesting this is automatically generated. if you are creating automated commits please remove the emojis, it isn't clear what they mean. I am not sure what `i` and `up` mean here, but it seems now like you were intending to say that "this was automated". In general I would recommend against using emojis to convey information, as they are very confusing and have multiple interpretations.

**Comment by DerekNonGeneric** [comment]
My bad about emoji in the first commit of this PR, I fell asleep before I could fix it. Since you had expressed interest in machine-reable issue titles, I thought that you would see value in machine-readable PR titles too. Since the PR title is usually derived from the first line of the commit message, those could be machine-readable as well. A pull request template going into detail about this would be amazing.

**Comment by DerekNonGeneric** [comment]
We need this merged to move forward on the crawler to pull off something like git patch otherwise the diff would be too chaotic to be workable&hellip;

**Pull request merged**